pkg/broker/ingress/ingress_handler.go

```diff
@@ -257,9 +257,14 @@ func (h *Handler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
 	}
 
 	reporterArgs := &ReportArgs{
-		ns:        brokerNamespace,
-		broker:    brokerName,
-		eventType: event.Type(),
+		ns:          brokerNamespace,
+		broker:      brokerName,
+		eventType:   event.Type(),
+		eventScheme: request.URL.Scheme,
```

Comment:
Again here, we should be properly checking if the connection is TLS, rather than using the Scheme as using the Scheme will always be empty, resulting in us always reporting http which is wrong